src/app/workflow/versions/versions.component.ts

```diff
@@ -245,4 +247,20 @@ export class VersionsWorkflowComponent extends Versions implements OnInit, OnCha
   trackBy(index: number, item: WorkflowVersion) {
     return item.id;
   }
+
+  showExecutionMetricsIcon(version: WorkflowVersion) {
```

Comment:
Nit, whether you make this a pipe or not, I suggest passing `version.metricsByPlatform` as the parameter, that way you can get rid of the 2 `version.` strings inside the method.